### ISSUE TITLE
Adding Distortions and Tactical Sense

### ISSUE THREAD

**Issue by CheraZakalwe** [Venomed/SkyforgeAelinetScraper#3]
Mahallo!!

Chera here. As per your request, our request: We would like to have a few columns added to the right of the current version.

We're thinking of something like that: TS     | A1 | A1R | A2 | A2R | A3 | A3R | ...
                                                            2600    X     X                        X 

Hope my little noob-formatting still works after posting this. We shall see.
EDIT: Alright it does not. The Xs are supposed to be the marking below the distortions....

The result will hopefully be the currently working prestige/credit/buildingmat overview, and then we can just copy the right side of the entire thing, starting from TS, and put that in a different spreadsheet, basically a raid-overview.

If possible, it would of course be nice to have the linqpad program output the result directly in 2 spreadsheets, in which case the 2 columns left of TS would be ID and name of the player.

I hope I'm making some sense here, if not, you know where to reach me, master Maverik :)

**Comment by Maverik** [comment]
Since there's far too much information, I'll dump it as a second table which can be used as is in second sheet. I'm still thinking about how to best do this but I have a starting point at least.

It appears this XHR lets me access all of dungeon information:

```
GET /api/game/stats/StatsApi:getAvatarStats/72061992084499827?csrf_token=XhhwTLCYbaZOnjCE HTTP/1.1
Host: eu.portal.sf.my.com
Connection: keep-alive
Pragma: no-cache
Cache-Control: no-cache
Accept: application/json, text/plain, */*
X-Requested-With: XMLHttpRequest
User-Agent: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.109 Safari/537.36
DNT: 1
Referer: https://eu.portal.sf.my.com/user/avatarstats/72061992084499827
Accept-Encoding: gzip, deflate, sdch
Accept-Language: en-GB
Cookie: xxx
```

To get personal stats, I'll need to handle various bits in here:

```
GET /user/avatar/72061992084499827 HTTP/1.1
Host: eu.portal.sf.my.com
Connection: keep-alive
Pragma: no-cache
Cache-Control: no-cache
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
Upgrade-Insecure-Requests: 1
User-Agent: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.109 Safari/537.36
DNT: 1
Referer: https://eu.portal.sf.my.com/wall/72061992084499827
Accept-Encoding: gzip, deflate, sdch
Accept-Language: en-GB
Cookie: xxx
```

XPATH is: `//*[@id="portalPageBody"]/div[3]/div/div[2]/div/div[2]` for the segment with stats (though hopefully i won't need this with Simplebrowser but just for info)

**Comment by Maverik** [comment]
Closed by 65a7d64ac98ab94422021d530e4989d3ec089d48
